KnownIssues.md

```diff
@@ -1,63 +1,31 @@
-The following list are known issues. The issue may contain a workaround or an associated Issue number.
+## Known Issues for Oracle WebLogic Server Deploy Tooling
 
-**ISSUE**:
-   The createDomain and updateDomain tools cannot target non-JRF product resources to dynamic clusters because
-   WebLogic WLST will not assign resources associated with extension or custom template user server groups
-   to dynamic clusters. Resources associated to JRF user server groups defined in the domain typedef
-   (i.e. JRF, RestrictedJRF) will be targeted to the dynamic cluster by the WDT tools using the FMW WLST function
-   applyJRF.
-
-   If you have only non-JRF user server groups targeted to a dynamic cluster, you will see the following message:
-
-    WLSDPLY-12238: Unable to target non-JRF template server groups for domain type <your domain typedef> to dynamic cluster(s).
-
-   You will not see this message if you have a mix of non-JRF and JRF user server groups targeted to the dynamic
-   cluster. WDT cannot detect if a user server group is associated to JRF, and therefore, whether the applyJRF will
-   target the user group resources to the dynamic cluster.
-
-**ACTION**:
+The following list are known issues. The issue may contain a workaround or an associated issue number.
```

Comment:
The following list are known issues. (awkward wording) ->Two choices: In the following list are known issues. or The following list contains known issues.